analytic_engine/src/table/data.rs

```diff
@@ -175,7 +175,7 @@ impl Drop for TableData {
 
 #[inline]
 fn get_mutable_limit(opts: &TableOptions) -> u32 {
-    opts.write_buffer_size / 8 * 7
+    opts.write_buffer_size * 5 / 8
```

Comment:
"/ 8 * 5" may be better.
The original code "/ 8 * 7" considered the problem of overflow.

Reply:
Maybe we should not hard code it but make it a config?